bin/migrate-files/migrate-files.py

```diff
@@ -39,27 +41,48 @@
     with open('last-file.pickle', 'rb') as file:
         starting_document_id = pickle.load(file)
 
-print('Starting from:', starting_document_id)
+print('Starting applications from:', starting_document_id)
+
+starting_planning_document_id = 0
+# Determine job resume status
+if os.path.isfile('last-planning-file.pickle'):
+    with open('last-planning-file.pickle', 'rb') as file:
+        starting_planning_document_id = pickle.load(file)
+    #ignore imported files before failure
+    starting_document_id = 999999
+
+print('Starting planning_review from:', starting_planning_document_id)
+
+starting_issue_document_id = 0
+# Determine job resume status
+if os.path.isfile('last-issue-file.pickle'):
+    with open('last-issue-file.pickle', 'rb') as file:
+        starting_issue_document_id = pickle.load(file)
+    #ignore imported files before failure
+    starting_document_id = 999999
+    starting_planning_document_id = 999999
+print('Starting issues from:', starting_issue_document_id)
 
 # Get total number of files
 
 cursor = conn.cursor()
 
 try:
```

Comment:
lets split this function into: transfer_application_documents, transfer_noi_documents, transfer_planning_review_documents